--- tomoe-0.6.0/macros/ruby.m4.orig	2007-06-11 03:05:42.000000000 +0200
+++ tomoe-0.6.0/macros/ruby.m4	2012-11-12 21:18:04.642203203 +0100
@@ -28,14 +28,15 @@
 
   changequote(<<, >>)
   for var_name in archdir sitearchdir CFLAGS LIBRUBYARG libdir \
-                  sitelibdir sitearchdir; do
-    rbconfig_tmp=`$rbconfig "print Config::CONFIG['$var_name']"`
+                  vendorlibdir vendorarchdir rubyhdrdir sitehdrdir arch; do
+    rbconfig_tmp=`$rbconfig "print RbConfig::CONFIG['$var_name']"`
     eval "rbconfig_$var_name=\"$rbconfig_tmp\""
   done
+  rbconfig_archhdrdir="$rbconfig_rubyhdrdir/$rbconfig_arch"
   changequote([, ])
 
-  RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_archdir "
-  RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_sitearchdir "
+  RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_archdir -I$rbconfig_rubyhdrdir -I$rbconfig_archhdrdir"
+  RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_vendorarchdir -I$rbconfig_vendorhdrdir"
   RUBY_CFLAGS="$RUBY_CFLAGS $rbconfig_CFLAGS "
   RUBY_LIBS="$rbconfig_LIBRUBYARG"
   RUBY_LDFLAGS="-L$rbconfig_libdir"
@@ -69,7 +69,7 @@
 
   AC_MSG_CHECKING([where to install Ruby binding])
   AC_CACHE_VAL([cv_ruby_archdir],
-               [cv_ruby_archdir="$rbconfig_sitearchdir"])
+               [cv_ruby_archdir="$rbconfig_vendorarchdir"])
   AC_ARG_WITH([ruby-bindingdir],
                 AS_HELP_STRING([--with-ruby-bindingdir=EXTDIR],
                                [install Ruby bindings in EXTDIR
@@ -80,13 +80,13 @@
 
   AC_MSG_CHECKING([where to install Ruby scripts])
   AC_CACHE_VAL([cv_ruby_sitedir],
-               [cv_ruby_sitedir="$rbconfig_sitelibdir"])
+               [cv_ruby_sitedir="$rbconfig_vendorlibdir"])
   AC_ARG_WITH([ruby-libdir],
                AS_HELP_STRING([--with-ruby-libdir=LIBDIR],
                               [install Ruby scripts for bindings in LIBDIR
                                (default is same as ruby's one)]),
                [ruby_libdir="$withval"],
-               [ruby_libdir="$cv_ruby_sitedir"])
+               [ruby_libdir="$cv_ruby_vendordir"])
   AC_MSG_RESULT([$ruby_libdir])
 
   AC_SUBST(ruby_bindingdir)
--- tomoe-0.6.0/bindings/ruby/tomoe-rb-char.c.orig	2007-06-06 02:44:38.000000000 +0200
+++ tomoe-0.6.0/bindings/ruby/tomoe-rb-char.c	2012-11-12 21:20:44.718863156 +0100
@@ -13,7 +13,7 @@
     if (NIL_P(xml)) {
         chr = tomoe_char_new();
     } else {
-        chr = tomoe_char_new_from_xml_data(RVAL2CSTR(xml), RSTRING(xml)->len);
+        chr = tomoe_char_new_from_xml_data(RVAL2CSTR(xml), RSTRING_LEN(xml));
     }
 
     G_INITIALIZE(self, chr);
